CHANGELOG.md

```diff
@@ -1,22 +1,172 @@
+    [pull/664]: https://github.com/rustwasm/wasm-pack/pull/664
+    [issue/639]: https://github.com/rustwasm/wasm-pack/issues/639  
+
 ## 🛠️ 0.8.1
 
 - ### 🤕 Fixes
 
   - **Check for "rustup" rather than ".rustup" when checking for wasm32 - [drager], [issue/613][pull/616]**
 
-      When we introduced support for non-rustup setups we did a check if the user was
-      using rustup or not. However, this check was too constrained and only covered
-      the most common cases, but it did not work for Docker setups.
+    When we introduced support for non-rustup setups we did a check if the user was
+    using rustup or not. However, this check was too constrained and only covered
+    the most common cases, but it did not work for Docker setups.
+
+    This PR addresses that and it now covers Docker setups as well!
+    When doing this fix we also found two other small issues which this PR also addresses.
+    The first is that we did not print the helpful error message when the wasm32 target
+    was not found and the other one was that it linked to the wrong section of the documentation.
 
-      This PR addresses that and it now covers Docker setups as well!
-      When doing this fix we also found two other small issues which this PR also addresses.
-      The first is that we did not print the helpful error message when the wasm32 target
-      was not found and the other one was that it linked to the wrong section of the documentation.
+    [issue/613]: https://github.com/rustwasm/wasm-pack/issues/613
+    [pull/616]: https://github.com/rustwasm/wasm-pack/pull/616
+
+  - **Handle version check failures - [drager], [issue/652], [issue/653] [pull/660]**
+
+    Every day, `wasm-pack` checks the crates.io API for the latest version number and lets the user know if their installation is out of date. Now, when these API calls fail, `wasm-pack` alerts the user of the failure and waits until the next day to make another call to crates.io.
+
+    [drager]: https://github.com/drager
+    [pull/660]: https://github.com/rustwasm/wasm-pack/pull/660
+    [issue/652]: https://github.com/rustwasm/wasm-pack/issues/652
+    [issue/653]: https://github.com/rustwasm/wasm-pack/issues/653
 
-      [issue/613]: https://github.com/rustwasm/wasm-pack/issues/613
-      [pull/616]: https://github.com/rustwasm/wasm-pack/pull/616
+  - **Add user agent for version check - [drager], [issue/651] [pull/658]**
+
+    crates.io requires tools to set a version check `User-Agent` header when requesting the latest version. Now, when `wasm-pack` performs an API request to crates.io, it sends `User-Agent: wasm-pack/0.9.0`.
+
+    [drager]: https://github.com/drager
+    [pull/658]: https://github.com/rustwasm/wasm-pack/pull/658
+    [issue/651]: https://github.com/rustwasm/wasm-pack/issues/651
+
+  - **Remove broken link from the README - [drager], [pull/635]**
+
+    [drager]: https://github.com/drager
+    [pull/635]: https://github.com/rustwasm/wasm-pack/pull/635 
```

Comment:
Hmm, were these released in 0.8.1? IIRC these wasn't released and should be released in 0.9? I might be wrong though.

Reply:
oh woops! yes i added these to the wrong section - good catch